web/src/main/java/no/nav/modiapersonoversikt/rest/persondata/PersondataFletter.kt

```diff
@@ -360,12 +360,67 @@ class PersondataFletter(val kodeverk: EnhetligKodeverk.Service) {
                 Persondata.SistEndret(
                     ident = it.registrertAv,
                     tidspunkt = it.registrert.value,
-                    system = it.systemkilde,
-                    kilde = it.kilde
+                    system = mapOmSystemKildeFraEndringerMetadata(it.systemkilde),
+                    kilde = mapOmKildeFraEndringerMetadata(it.kilde)
                 )
             }
     }
 
+    private fun mapOmSystemKildeFraEndringerMetadata(system: String): String {
+        return when (system) {
+            "FREG" -> "folkeregisteret"
+            "BD03" -> "bruker"
+            "srvpersonopplysnin" -> "srvpersonopplysning"
+            "PP01" -> "NAV"
+            "BD06" -> "NAV"
+            "FS22" -> "NAV"
+            "personopplysninger-api" -> "personopplysninger-api"
+            "srvperson-forvalter" -> "NAV"
+            "IT00" -> "NAV"
+            "srvPdl-Web" -> "srvPdl-Web"
+            "BI00" -> "NAV"
+            "pdl-web" -> "pdl-web"
+            "SrvOppgRobotNOP" -> "NAV"
+            else -> system
+        }
+    }
+
+    private fun mapOmKildeFraEndringerMetadata(kilde: String): String {
+        return when (kilde) {
+            "FREG" -> "folkeregisteret"
+            "innbygger" -> "bruker"
+            "KILDE_DSF" -> "DSF"
+            "Matrikkelen" -> "statens kartverk"
+            "TPS" -> "TPS"
+            "BRUKER SELV" -> "bruker"
+            "Bruker selv" -> "bruker"
+            "folkeregistermyndigheten" -> "folkeregistermyndigheten"
+            "utlendingsdirektoratet" -> "utlendingsdirektoratet"
+            "SKATTEETATEN" -> "skatteetaten"
+            "tingretten" -> "tingretten"
+            "saksbehandler" -> "saksbehandler"
```

Comment:
Noen av disse mapper bare til lik verdi, er det for å eksplisitt vise at det ordet er håndtert?

Reply:
Ja, må avklares om det er nødvendig å mappe de i det hele tatt når det blir samme verdi eller om det er ønskelig å ha de der for å vise at dette er kilder man kan forvente å få